### ISSUE TITLE
Wrong arguments in call to retrieve_movie_info_from_server function

### ISSUE THREAD

**Issue by pilarnavarro** [ocean-data-factory-sweden/kso-utils#81]
The arguments provided in the function call to `retrieve_movie_info_from_server` within the `choose_footage` function in the `widget.py` script do not correspond to the parameters declared in the function inside the `movie_utils.py` file.

**Comment by jannesgg** [comment]
@pilarnavarro Have you pulled in the latest changes in the dev branch of kso_utils?

On my side I cannot find any `choose_footage` function used in `movie_utils.py `

It is only called in `widgets.py` and in tutorial 6 as:

source = kso_widgets.choose_footage(
    project,
    pp.db_info,
    project.movie_folder if project.movie_folder not in [None, "None"] else ".",
    "custom footage",
)

Which is the same parameters as in `widgets.py`:

choose_footage(project: Project, db_info_dict: dict, start_path: str = ".", folder_type: str = "")

**Comment by pilarnavarro** [comment]
You are right, but I am referring to the function `retrieve_movie_info_from_server` (which is defined in `movie_utils.py`) and is called with incorrect arguments in the definition of `choose_footage`.

**Comment by jannesgg** [comment]
@victor-wildlife Is this perhaps related to changes that affect `db_info_dict`? I see that the `db_info_dict` argument has been swapped out for `server_info` in `movie_utils.py`?

**Comment by pilarnavarro** [comment]
Yes, that is one of the mismatches. But, most importantly, the `db_connection` parameter expected by `retrieve_movie_info_from_server` is not given.

**Comment by victor-wildlife** [comment]
I am going through the instances when we call for the "db_info_dict" and updating them. Will report back when I finish

**Comment by jannesgg** [comment]
@victor-wildlife Is this issue resolved now after the pull request?

**Comment by victor-wildlife** [comment]
@jannesgg yes, this issue will be fixed after the #105 pull request.
I have created another issue as I noticed that the current code to choose movies from AWS generates a temporary link for each of the movies available and then the user selects the movie of interest. To maximise our resources we should first select the movies of interest and then generate a temporary HTTP link for those movies only.
